client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -44,6 +47,10 @@ module.exports = React.createClass( {
 		};
 	},
 
+	isMobileTouchDevice() {
+		return ! isDesktop() || hasTouch();
```

Comment:
We might be fine with just the hasTouch check. I testing the image editor at smaller screen sizes and it looks like it still works OK.